config/slack-github-issues.json

```diff
@@ -24,7 +24,8 @@
       "channelNames": [
         "general-talk",
         "hub",
-        "questions"
+        "questions",
+        "newcomers"
```

Comment:
This needs to be in alphabetical order relative to the other `channelNames`. (Or, if you think it's ready, get rid of `channelNames` to turn it on for _every_ slack channel.)